data/DALTON/basicDALTON-2015/Trp_polar_abalnr.dal

```diff
@@ -0,0 +1,63 @@
+BASIS
+STO-3G
+tryptophan static polarizability
```

Comment:
It's a very minor time commitment, but we should probably hold off anyway until after making some decisions about the future of the test calculations.